doc/OnlineDocs/pyomo_overview/math_modeling.rst

```diff
@@ -72,8 +72,8 @@ solvers to analyze a model introduces additional complexities.
 
 
 Pyomo is an AML that extends Python to include objects for mathematical
-modeling. [PyomoBookI]_, [PyomoBookII]_, and [PyomoJournal]_ compare
-Pyomo with other AMLs.  Although many good AMLs have been developed for
+modeling. [PyomoBookI]_, [PyomoBookII]_, [PyomoBookIII]_, and [PyomoJournal]_ 
```

Comment:
Is there a reason to leave the references to the 1st and 2nd editions here?

Reply:
Okay, so, there are multiple places where previous versions of the book were referenced, and I am hesitant to change them because I do not know what things are or are not included in the 3rd edition that weren't in the 1st or 2nd edition. This is one of those times where I just don't know.